### ISSUE TITLE
POST URL

### ISSUE THREAD

**Issue by khargy** [ahenry91/pebble_url_launcher#2]
I needed to be able to send a POST URL and not a GET. I forked your project and updated the code if you are interested.

**Comment by andymitchhank** [comment]
I'm looking into a better way than using a URL variable to handle this. It will probably be a dropdown selection on the config page.

**Comment by khargy** [comment]
Yep I figured that make sense, it didn't seem like I was able to modify the
config page, so I was just using this as a proof of concept for myself to
see if the URL not being a POST was why it wasn't working for me.

Ken
